WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -624,7 +635,7 @@ public static void addOrUpdatePosts(final ReaderTag tag, ReaderPostList posts) {
                 stmtPosts.bindString(16, post.getFeaturedImage());
                 stmtPosts.bindString(17, post.getFeaturedVideo());
                 stmtPosts.bindString(18, post.getPostAvatar());
-                stmtPosts.bindLong  (19, post.timestamp);
+                stmtPosts.bindDouble(19, post.timestamp);
```

Comment:
> but... when you write near-to-perfect code, it's easier for the reviewer to point fingers on stuff that don't really matter

Hah! Thanks for the ego boost, but after you've been here for a while you may feel otherwise ;)